inst/rmarkdown/templates/nutrition/skeleton/skeleton.Rmd

```diff
@@ -72,9 +75,11 @@ survey_data$height <- as.numeric(survey_data$height)
 survey_data <- filter(survey_data, age_months > 0, age_months < 60)
 
 survey_data$age_group <- age_categories(survey_data$age_months, breakers = c(6, 18, 30, 42, 54, 60))
+survey_data$age_24months <- age_categories(survey_data$age_months, breakers = c(6, 24, 60))
 
 # drop unused group
 survey_data$age_group <- forcats::fct_drop(survey_data$age_group, "60+")
+survey_data$age_24months <- forcats::fct_drop(survey_data$age_24months, "60+")
 ```
 
 
```

Comment:
Zhian has updated to be able to create age pyramid as proportion of total population.
However we will need to create another function to allow weighted proportions!